src/cogent3/app/composable.py

```diff
@@ -1311,7 +1311,7 @@ def is_composable(obj):
 def _apply_to(
     self,
     dstore,
-    get_data_source: callable = get_data_source,
+    id_from_source: callable = get_data_source,
```

Comment:
@GavinHuttley 
Sorry, I forgot to do this change in the previous PR.
Is it good to rename `get_data_source` function, too.

Reply:
Because, as you renamed the argument, we don't get source. We get id from source in that function.

Reply:
I agree, but I think that will require more time than I have right now.